modules/users/client/services/password-validator.client.service.js

```diff
@@ -11,7 +11,7 @@ angular.module('users').factory('PasswordValidator', ['$window',
         return result;
       },
       getPopoverMsg: function () {
-        var popoverMsg = 'Please enter a passphrase or password with greater than 10 characters, numbers, lowercase, upppercase, and special characters.';
+        var popoverMsg = 'Please enter a passphrase or password with more than 10 characters, numbers, lowercase, uppercase, and special characters.';
```

Comment:
Thanks for catching this @Meistercoach83! @mleanos is right, the popover was intended to give the user instructions for the password requirements when they click into the form field.